FabricObserver/Observers/AppObserver.cs

```diff
@@ -2755,52 +3063,67 @@ private async Task SetDeployedApplicationReplicaOrInstanceListAsync(Uri applicat
                 }
 
                 XmlNodeList childNodes = sNode.ChildNodes;
-                XmlNode rgNode = null;
 
                 foreach (XmlNode node in childNodes)
                 {
-                    if (node.Name == "Policies")
+                    if (node.Name != ObserverConstants.PoliciesNodeName)
                     {
-                        foreach (XmlNode polNode in node.ChildNodes)
+                        continue;
+                    }
+
+                    foreach (XmlNode polNode in node.ChildNodes)
+                    {
+                        if (polNode.Name != ObserverConstants.RGPolicyNodeName)
```

Comment:
Was it discussed with Milan that we will only go at the code package level?

Because there are 2 policies that we can define, one at the service package level and another at the code package level.

https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-resource-governance#specify-resource-governance

https://learn.microsoft.com/en-us/dotnet/api/system.fabric.management.servicemodel.servicepackageresourcegovernancepolicytype?view=azure-dotnet

https://learn.microsoft.com/en-us/dotnet/api/system.fabric.management.servicemodel.resourcegovernancepolicytype?view=azure-dotnet

Reply:
Only CodePackage specification is supported. ServicePackageResourceGovernancePolicy element is generally not used to specify RG limits (it is unclear when from documentation when to use one or the other or, more likely, that both are required) and there will generally always be RG limits specified for CodePackages. At least, this is how AppObserver will expect the game to be played for now. Will do more research on this in the meantime.